Wrapping/Generators/Python/itkConfig.py.in

```diff
@@ -60,35 +60,35 @@ def itkFormatWarning(msg, *a, **kwa):
 
 
 # Redefine the format of the warnings
-warnings.formatwarning = itkFormatWarning
+warnings.formatwarning = _itk_format_warning
 
 
-def normalized_path(relative_posix_path):
+def _normalized_path(relative_posix_path):
     if relative_posix_path != "None":
         file_dir = os.path.split(__file__)[0]
         relative_path = relative_posix_path.replace("/", os.sep)
         return os.path.normpath(os.path.join(file_dir, relative_path))
 
 
 # swig_lib: location of the swig-generated shared libraries
-swig_lib = normalized_path("@CONFIG_PYTHON_SWIGLIB_DIR@")
+swig_lib = _normalized_path("../../../lib")
```

Comment:
This maybe should not have been auto-replaced? Same for the few instances below.

Reply:
OH!  Great catch.